### PR TITLE
fix binomial. Closes #407

### LINKED ISSUE #407: Binomial in randgraphs is causing tests to fail
@Ward9250 - see https://travis-ci.org/JuliaGraphs/LightGraphs.jl/builds/150828065#L332-L352 - specifically, I think that Binomial is being called with (Float64, Float64) when it should be called with (Int, Float64). Could you check? Thanks.

### PR DESCRIPTION
**Pull request by CarloLucibello**
#407

### CODE REVIEW

File: src/randgraphs.jl [hunk 1]
```diff
@@ -42,12 +42,12 @@ or the  `Graph(nv, ne)` constructor, which randomly select `ne` edges among all
 edges.
 """
 function erdos_renyi(n::Integer, p::Real; is_directed=false, seed::Integer=-1)
-    m = is_directed ? n*(n-1) : div(n*(n-1),2)
+    m = Int(is_directed ? n*(n-1) : div(n*(n-1),2))
```
Comment (andreasnoack) [inline src/randgraphs.jl]: The `Int` here shouldn't be necessary

File: src/randgraphs.jl [hunk 2]
```diff
@@ -529,9 +529,9 @@ function stochastic_block_model{T<:Real}(c::Matrix{T}, n::Vector{Int}; seed::Int
         for b=a:K
             @assert a==b? c[a,b] <= n[b]-1 : c[a,b] <= n[b]   "Mean degree cannot be greater than available neighbors in the block."
 
-            m = a==b ? n[a]*(n[a]-1)/2 : n[a]*n[b]
+            m = Int(a==b ? n[a]*(n[a]-1)/2 : n[a]*n[b])
```
Comment (andreasnoack) [inline src/randgraphs.jl]: It would be better to use integer division. Then the elements would never be promoted to floats.

Comment (CarloLucibello) [inline src/randgraphs.jl]: right

File: src/randgraphs.jl (hunk 1 above)
Comment (CarloLucibello) [inline src/randgraphs.jl]: right

**Pull request merged**